package.json

```diff
@@ -9,9 +9,10 @@
     "lint": "biome lint",
     "check": "biome check --write",
     "check:ci": "biome check",
-    "prepare": "husky && bob build",
+    "prepare": "husky && bob build && yarn build:plugin",
```

Comment:
what about test command? Will it work without pre building?

Reply:
I changed it so it tests against the source files directly now